### PR TITLE
Native time picker revision

### PR DESCRIPTION
**Pull request by lisavo7**

### CODE REVIEW

**Review by debra-lonsdorf — DISMISSED** [review]

File: _checklist-native/calendar-date-picker.md [hunk 1]
```diff
@@ -79,10 +78,12 @@ settings:
 - **SwiftUI**
   - Use view modifier `accessibilityElement(children: .combine)` to merge the child accessibility element’s properties into the new accessibilityElement.
 
-### State 
+### State
+- By default, disabled or unavailble dates are announced as dimmed
```
Comment (jzoo) [inline _checklist-native/calendar-date-picker.md:82]: spelling updated needed for unavailable

File: _checklist-native/pagination-control.md [hunk 2]
```diff
@@ -24,86 +24,193 @@ screenreader:
...
 ---
 
+## iOS
+### Developer notes
 
-## Developer notes
+- A control that displays a horizontal series of dots, each of which corresponds to a screen/page or slide
+- There are a variety of implementation alternatives for a page control:  
+  - The screen reader focus may remain on the page control when it is focused and the user interacts with it.  Custom actions can be implemented on the dots so that the screen reader user can swipe rt or left past the page control to navigate through the screen.  If there are many pages, having to swipe through all of them to get to the rest of the screen may not be a good user experience
+  - Swiping rt or left to change the pages while focus is on the page control is acceptable, if there are not too many pages.  Index of the page (2 of 4) must be announced
```
Comment (jzoo) [inline _checklist-native/pagination-control.md:44]: "rt" is that supposed to be right, if so can we udpate

File: _checklist-native/pagination-control.md [hunk 3]
```diff
@@ -24,86 +24,193 @@ screenreader:
...
+  - step 2: Add the `FLAG_REQUEST_ACCESSIBILITY_BUTTON` flag in an AccessibilityServiceInfo object's `android:accessibilityFlags` attribute
+  - step 3: To have a custom service register for the button's custom action callbacks, use `registerAccessibilityButtonCallback()`
+
+- **Jetpack Compose**
+  - List of custom accessibility actions can be defined relatively easily in compose compared to Views using customActions. 
+  - Example: `modifier = Modifier.semantics { customActions = listOf(CustomAccessibilityAction(label = "", action = { true }))}`
+
+### Announcements
+  - "Label, page three of three, List, out of grid pager”   If focus is on pagination dots and they are not interactive
+  - “Text, images, in horizontal pager”  When focus is in body of page.  Swipe up and rt (or left) with two fingers to change page
```
Comment (jzoo) [inline _checklist-native/pagination-control.md:215]: change "rt" to right

File: _checklist-native/pagination-control.md [hunk 4]
```diff
@@ -24,86 +24,193 @@ screenreader:
...
+  - “ Images, text, page 1 of 3, list, double tap to activate”  Swipe rt or left to navigate through slides and announcement includes meaningful content on the page and text, plus the index.  Pagination dots are not focusable.
```
Comment (jzoo) [inline _checklist-native/pagination-control.md:216]: change "rt" to right

File: _checklist-native/pagination-control.md [hunk 5]
```diff
@@ -24,86 +24,193 @@ screenreader:
...
+- **SwiftUI**
+  - For general focus management that impacts both screen readers and non-screen readers, use the property wrapper `@FocusState` to assign an identity of a focus state.
+    - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:)` to assign focus on a view with `@FocusState` as the source of truth.
+    - Use the property wrapper `@FocusState`in conjunction with the view modifier `focused(_:equals:)` to assign focus on a view, when the view is equal to a specific value.
+  - If necessary, use property wrapper `@AccessibilityFocusState` to assign identifiers to specific views to manually shift focus from one view to another as the user interacts with the screen with VoiceOver on.
+
+### Announcements
+  - "Page three of three, adjustable, swipe or down with one finger to adjust the value"  Focus is on pagination dots, custom actions have been implemented and the dots are interactive
+  - “ Images, text, page 1 of 3, button”  Swipe rt or left to navigate through slides and announcement includes meaningful content on the page and text, plus the index.  Pagination dots are not focusable.
```
Comment (jzoo) [inline _checklist-native/pagination-control.md:113]: change "rt" to right

File: _checklist-native/pagination-control.md [hunk 6]
```diff
@@ -24,86 +24,193 @@ screenreader:
...
+  - `UIAccessibilityContainer` protocol: Have a table of elements that defines the reading order of the elements.  
+- **SwiftUI**
+  - For general focus management that impacts both screen readers and non-screen readers, use the property wrapper `@FocusState` to assign an identity of a focus state.
+    - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:)` to assign focus on a view with `@FocusState` as the source of truth.
+    - Use the property wrapper `@FocusState`in conjunction with the view modifier `focused(_:equals:)` to assign focus on a view, when the view is equal to a specific value.
+  - If necessary, use property wrapper `@AccessibilityFocusState` to assign identifiers to specific views to manually shift focus from one view to another as the user interacts with the screen with VoiceOver on.
+
+### Announcements
+  - "Page three of three, adjustable, swipe or down with one finger to adjust the value"  Focus is on pagination dots, custom actions have been implemented and the dots are interactive
```
Comment (jzoo) [inline _checklist-native/pagination-control.md:112]: i think we are missing the word "up", so update to, "swipe up or down...."

**Review by jzoo — CHANGES_REQUESTED** [review]
i have a few items, a few spelling changes and updating "rt" to "right". we typically give the guidance in A11Y to avoid abbreviations when possible so i think we need to follow that here as well by changing "rt" to "right"

**Review by jzoo — APPROVED** [review]
looks good

**Pull request merged**